activities.json

```diff
@@ -1154,6 +1154,18 @@
     "title": "enterkeyhint attribute",
     "url": "https://html.spec.whatwg.org/multipage/interaction.html#attr-enterkeyhint"
   },
+  {
+    "ciuName": null,
```

Comment:
This should probably be `"mdn-api_htmlelement_inert"`.

(I can't do suggested changes anymore -- not sure if that's because the PR is closed or because the branch is deleted.)